tests/ndarrays/test_ndarrays.c

```diff
@@ -265,6 +353,35 @@ int32_t test_indexing_int8(void)
     return (0);
 }
 
+int32_t test_indexing_int8_order_f(void)
+{
+    int8_t m_1[] = {1, 6, 6, 2, 4, 0, 1, 5,
+                    7, 1, 3, 5, 3, 6, 4, 9,
+                    7, 6, 7, 1, 7, 7, 4, 4,
+                    7, 1, 9, 0, 1, 2, 4, 5,
+                    6, 1, 7, 3, 1, 5, 9, 6};
+    int64_t m_1_shape[] = {5, 8};
+    t_ndarray x;
+    int32_t index;
+    int32_t c_index;
+    int8_t value;
+    int8_t c_value;
+
+    x = array_create(2, m_1_shape, nd_int8, false, order_f);
+    memcpy(x.raw_data, m_1, x.buffer_size);
+    // testing the index [3, 2]
+    index = 3 * x.strides[0] + 2 * x.strides[1];
+    c_index = 13;
+    my_assert(index , c_index, "testing the strides");
+    my_assert(get_index(x, 3, 2) , c_index, "testing the indexing function");
```

Comment:
This testing is good, but as you need multiple functions anyway, why not test a few more indexes?